docs/data/migration/usage-with-material-ui-v5-v6/usage-with-material-ui-v5-v6.md

```diff
@@ -50,16 +49,15 @@ export default defineConfig({
 
 ### Next.js Pages router
 
-With [Next.js App router](https://nextjs.org/docs#app-router-vs-pages-router), you don't need to do anything.
+With [Next.js App Router](https://nextjs.org/docs#app-router-vs-pages-router), you don't need to do anything.
 
-With Next.js Pages router, update Next.js configuration to pass the `require` condition and transpile MUI X packages:
-
-```ts
-// next.config.js
```

Comment:
Nop, it has to be .mjs: https://nextjs.org/docs/pages/api-reference/config/next-config-js#ecmascript-modules